xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -318,6 +318,9 @@ void CXBMCApp::onResume()
     appPower->ResetShutdownTimers();
   }
 
+  if (CServiceBroker::GetAppMessenger())
```

Comment:
This check makes only sense, when refcounting would be involved. Therefore assign the GetAppMessenger to a local variable and then use this for sending the message?

e.g. mymessenger = CServiceBroker::GetAppMessenger();
if (mymessenger)
{
}

Right?

Reply:
Good, use a variable